### PR TITLE
modified the report, added reference

### PR DESCRIPTION
**Pull request by Jing-19**

### CODE REVIEW

File: report/save_the_earth_model.ipynb [hunk 1]
```diff
@@ -116,7 +118,7 @@
    "source": [
     "## Analysis\n",
     "\n",
-    "Data was split with 80% partitioned as training data and 20% as test data. For model building, we have chosen KNeighborsRegressor (KNN) from DummyRegressor,Ridge, SVR, as we have the highest $\\text{R}^2$ score for KNN. The hyperparameter $K$ was chosen using 10-fold cross validation with $R^2$ as the regression metric."
+    "Out data was split into training (80%) and test (20%) set. We opted for the KNeighborsRegressor (KNN) from a pool of prospecive models - DummyRegressor,Ridge, SVR - givens its superior predictive accuracy reflected in the highest $\\text{R}^2$ score. The hyperparameter $K$ was tuned using 10-fold cross validation, guided by $R^2$ as the regression metric. Our analytical environment was powered by Python programming language {cite}Python along with the serveral packages: Pandas {cite}mckinney-proc-scipy-2010, Altair {cite}altair, scikit-learn {cite}scikit-learn, Matplotlib {cite}bisong2019matplotlib. To perform our analysis and create this report, the code  can be found here: https://github.com/UBC-MDS/DSCI_522_Group-11_Save-The-Earth/tree/main."
```
Comment (tonyshumlh) [inline report/save_the_earth_model.ipynb:121]: Out -> Our, prospecive -> prospective, serveral -> several

File: report/save_the_earth_model.ipynb [hunk 2]
```diff
@@ -63,7 +63,9 @@
...
    "source": [
-    "Here we attempt to build a prediction model using the k-nearest neighbours algorithm which can use energy consumption and energy generation measurements to predict CO2 emission of certain country of next year. Our final prediction model perform pretty well on unseen test dataset, with $\\text{R}^2$ of 0.975 and an overall accuracy calculated to be 0.976. However, the model predict CO2 emission by finding the existing cases in the training data set which is most similiar to unseen data, thus, if there is a case in unseen data set of which measurements are beyond the ranges in training data set (ie. massive increase of energy usage or energy efficiency increase or new type of clean energy), then the prediciton might not be accurate, thus we recomment continuing study to improve this prediction model."
+    "Here we attempt to build a prediction model employing the k-nearest neighbours algorithm, designed to leverage energy consumption and energy generation measurements to predict CO2 emissions of a country. Our model's performance on the unseen tst dataset is quite commendable, as reflected by an $\\text{R}^2$ of 0.975 and an overall accuracy of 0.976. \n",
```
Comment (tonyshumlh) [inline report/save_the_earth_model.ipynb:66]: tst: test, trianing: training, predicitive: predictive, reserach: research

File: report/save_the_earth_model.ipynb (hunk 1 above)
Comment (Jing-19) [reply report/save_the_earth_model.ipynb]: done!

File: report/save_the_earth_model.ipynb [hunk 3]
```diff
@@ -79,9 +81,9 @@
...
     "\n",
-    "Our project aims to estimate a machine learning model to use energy consumptions per capita to predict CO2 Emission per capita of a country. Our model can be a powerful tool for raising public awareness of the impact of energy consumption on CO2 emission and international agreements on emission reductions. We are hoping that our findings will encourage sustainable behavior, such as reducing energy consumption or opting for green energy alternatives (International Energy Agency, 2018). \n"
+    "Our project aims to estimate a machine learning model that utilizes per capita energy comsumption data to predict per capita CO2 emission for a country. Our model could serve as a potent instrument in raising public awareness about the correlation bewteen energy consumption and CO2 emission, and in the context of international international agreements on emission reductions. We aspire that our research will promote sustainable behavior, such as reducing energy consumption or choosing greener energy alternatives {cite}energy2018co2. \n"
```
Comment (tonyshumlh) [inline report/save_the_earth_model.ipynb:86]: comsumption: consumption, bewteen: between

File: report/save_the_earth_model.ipynb [hunk 4]
```diff
@@ -102,7 +104,7 @@
     "\n",
     "## Data\n",
     "\n",
-    "The data set that was used in this project is from World Bank via GAPMINDER.ORG, which is an independent Swedish foundation with no political, religious or economic affiliations and the link can be found https://www.gapminder.org/\n",
+    "The data set that was used in this project is from World Bank via GAPMINDER.ORG, which is an independent Swedish foundation with no political, religious or economic affiliations and the link can be found https://www.gapminder.org/. Each row in the data set represents a comsumption of a certain energy for a certain country at a given year.\n",
```
Comment (tonyshumlh) [inline report/save_the_earth_model.ipynb:107]: comsumption: consumption

**Review by tonyshumlh — APPROVED** [review]
Tested. Looks good to me

**Review by Aishwarya120111 — APPROVED** [review]

**Pull request merged**